### PR TITLE
Maybe mount /etc/machine-id

### PR DESCRIPTION
**Pull request by loveshack**

### CODE REVIEW

**Comment by reidpr** [comment]
Addresses #1050.

**Comment by reidpr** [comment]
Hello @loveshack; I pushed a style tweak and some documentation corrections. If it looks good to you, I'll merge.

File: doc/ch-run_desc.rst [hunk 1]
```diff
@@ -100,31 +109,30 @@ In addition to any directories specified by the user with :code:`--bind`,
...
-    :code:`/home/$USER` by default. This is accomplished by mounting a new
-    :code:`tmpfs` at :code:`/home`, which hides any image content under that
-    path. If :code:`--no-home` is specified, neither of these things happens
-    and the image's :code:`/home` is exposed unaltered.
+  * :code:`/etc/hosts`
+  * :code:`/etc/machine-id`
+  * :code:`/etc/resolv.conf`
+  * :code:`/var/lib/hugetlbfs` at guest :code:`/var/opt/cray/hugetlbfs`
+  * :code:`/var/opt/cray/alps/spool`
+  * :code:`$PREFIX/bin/ch-ssh` at guest :code:`/usr/bin/ch-ssh`
```
Comment (heasterday) [inline doc/ch-run_desc.rst:127]: Do we already document why we mount these files/directories? If not, I think that would be helpful for some of the less obvious ones.

I'm thinking these could use some explanation:
 * `/etc/machine-id`
 * `/var/lib/hugetlbfs`
 * `/var/opt/cray/alps/spool`
 * `$PREFIX/bin/ch-ssh`
 
 Thoughts?

**Comment by heasterday** [comment]
Tested this locally and it looks good, only one minor comment (which may be a separate PR).

File: doc/ch-run_desc.rst (hunk 1 above)
Comment (reidpr) [reply doc/ch-run_desc.rst:127]: (1) No (2) I agree

**Pull request merged**